src/Compilers/CSharp/Test/Emit/CodeGen/UnsafeTests.cs

```diff
@@ -5072,12 +5072,9 @@ public static class FixableExt
             var compVerifier = CreateCompilationWithMscorlib46(text, options: TestOptions.UnsafeReleaseExe);
 
             compVerifier.VerifyDiagnostics(
-                // (6,25): error CS9385: The given expression cannot be used in a fixed statement
-                //         fixed (int* p = new Fixable(1))
-                Diagnostic(ErrorCode.ERR_ExprCannotBeFixed, "new Fixable(1)").WithLocation(6, 25),
-                // (6,25): error CS0122: 'FixableExt.GetPinnableReference(Fixable)' is inaccessible due to its protection level
+                // (6,25): error CS8385: The given expression cannot be used in a fixed statement
```

Comment:
The problem is it currently gives *no* hints. Should this diagnostic also mention that the method `GetPinnableReference` could be called if it were found, but that it must be non-static, what its signature should be, and that it could be either an instance or extension method? I think this PR is the wrong place to start educating users about the new feature.